Zico.Training.SpecUnitRemover/Zico.Training.SpecUnitRemover.App/ContentRemover.cs

```diff
@@ -2,6 +2,9 @@
 {
     public class ContentRemover
     {
+        private static int _endParenthesisIndex;
```

Comment:
on this project this won't be a problem, but do you know the true implications of marking this as static?